docker/Dockerfile.worker-gpu

```diff
@@ -0,0 +1,31 @@
+FROM ufoym/deepo:all-py36
+
+# setup user/mail
+ARG name
+ARG email
+RUN mkdir ~/.openmined && \
+    echo '{"email": "'${email}'", "name": "'${name}'"}' > ~/.openmined/whoami.json
+
+# install libssl
+RUN apt-get update &&\
+    apt-get install -y libssl-dev
+
+# install requirements
+COPY ./requirements.txt /requirements.txt
+RUN pip install -r requirements.txt
+
+
+# jupyter
+RUN pip install jupyter
+COPY ./run_jupyter.sh /run_jupyter.sh
+COPY ./jupyter_notebook_config.py /root/.jupyter/
+RUN jupyter nbextension enable --py --sys-prefix widgetsnbextension && \
+    python -m ipykernel.kernelspec
+
+
+ENV PASSWORD=tf
```

Comment:
Is there a reason we're setting a default password for the jupyter server? Would it be easier for user if there wasn't a password set at all? If we do need to set a default password, it'd be nice if it was documented in the README.md.

Reply:
The newer jupyter versions default to using a login prompt with password. In order to get rid off the login screen and passwords, you can add `c.NotebookApp.token = ""` to `/etc/jupyter/jupyter_notebook_config.py`.

Reply:
I usually like to have a passwd for security reason. Say, you spawn a worker onto the cloud, then without a passwd, anybody that figure out the IP of the instance can access the notebook. 

I get ride of it for the time being following @johannes69ki comment